packages/shared/lib/services/sync/manager.service.ts

```diff
@@ -40,13 +40,23 @@ export interface CreateSyncArgs {
     environmentId: number;
     sync: IncomingFlowConfig;
     syncName: string;
-    variant: string;
+    syncVariant: string;
 }
 
 const logger = getLogger('sync.manager');
 
 export class SyncManagerService {
-    public async createSyncForConnection(connectionId: number, logContextGetter: LogContextGetter, orchestrator: Orchestrator): Promise<void> {
+    public async createSyncForConnection({
+        connectionId,
+        syncVariant,
+        logContextGetter,
+        orchestrator
+    }: {
+        connectionId: number;
+        syncVariant: string;
+        logContextGetter: LogContextGetter;
+        orchestrator: Orchestrator;
+    }): Promise<void> {
```

Comment:
unrelated but keeping consistent with the other function `createSyncForConnections` with a `s`